scan/context.go

```diff
@@ -121,26 +123,39 @@ func (s *Scanner) getContextFromReader(r io.Reader) (err error) {
 }
 
 func (s *Scanner) getContextFromRegistry(ctx context.Context, registryArtifact string) (err error) {
```

Comment:
How do we easily verify it?
I'm a bit confused by the command description... Do we have an example for `scan`?
![image](https://user-images.githubusercontent.com/8113721/163350692-27399028-3ff5-4b7d-9ffe-357efb27a090.png)

Reply:
I guess the scan module might give you an impression that the corresponding command is `scan`. It's also used in `download` command. To test the code, you can sync the code and run 
```
# build the binary into bin folder
make build
# download the oci artifact to default temp folder
cd bin
./acb download oci://binducentralus.azurecr.io/acb-oras-sample:arm
# verify the downloaded file
tree temp/arm
```

Reply:
Ok cool, it works! Thank you!

Reply:
Forgot to mention, here is the command to test authentication with `--credential` parameter

```
./acb download --credential '{"registry":"binducentralus.azurecr.io","username":"binducentralus","userNameProviderType":"opaque","password":"replace-the-password","passwordProviderType":"opaque"}' oci://binducentralus.azurecr.io/acb-oras-sample:arm
```